src/sqlast/mod.rs

```diff
@@ -346,6 +324,7 @@ impl ToString for SQLWindowFrameBound {
 }
 
 /// A top-level statement (SELECT, INSERT, CREATE, etc.)
+#[allow(clippy::large_enum_variant)]
```

Comment:
I see that clippy points at `SQLStatement::SQLUpdate` as being a "large variant" after this change. I don't understand why exactly, but I think going forward we might extract individual statement types making this a moot point.